public/locales/es/settings.json

```diff
@@ -2,6 +2,6 @@
   "language": "Lenguaje",
   "preferences": "Preferencias",
   "signed-approvals": "Habilitar aprobaciones firmadas",
-  "slippage-tolerance": "Slippage tolerance",
+  "slippage-tolerance": "Tolerancia de deslizamiento",
```

Comment:
I'm not so sure about this one. How other protocols call the slippage on ES? 
Maybe `Tolerancia de Slippage` would be better, to just have it as an english word,  wdyt?

Reply:
1inch:
![image](https://user-images.githubusercontent.com/287189/169670651-738effc7-919a-49f9-acff-c3b44b1527bc.png)

uniswap:
![image](https://user-images.githubusercontent.com/287189/169670656-0b51ba66-2262-4451-bd10-74fbea312279.png)


As a native spanish speaker, `Tolerancia de Slippage` sounds horrible, sorry. 🤷‍♂️

Reply:
I'm a native speaker too lol, it is really weird to translate everything, there are words that shouldn't be translated on my opinion, "tolerancia de deslizamiento" makes no sense in spanish, since slippage is a really native word in this case in crypto, I hope somebody finds a better wording some day haha